stqry.js

```diff
@@ -38,6 +89,10 @@ window.stqry = {
     set: function (changeset, callback, customKey) {
       var storageKey = customKey || STORAGE_KEY
       
+      if (window.stqryRuntime === "ReactNative") {
```

Comment:
@PanBachynskyi @barnaclebarnes this is how we detect that the code is running in our React Native app. Are we happy with the naming? It will be hard to change this once we ship. This property is only set for React Native, when it's not set we assume the runtime is Mobile Web.